tests/test_filling_fields.py

```diff
@@ -36,405 +34,385 @@
-        self.assertFalse(self.person.unhappy)
+        assert isinstance(person.unhappy, bool)
+        assert person.unhappy is False
 
 
-class DateFieldsFilling(FieldFillingTestCase):
+class TestDateFieldsFilling():
 
-    def test_fill_DateField_with_a_date(self):
+    def test_fill_DateField_with_a_date(self, person):
         birthday_field = models.Person._meta.get_field('birthday')
-        self.assertIsInstance(birthday_field, fields.DateField)
-        self.assertIsInstance(self.person.birthday, date)
+        assert isinstance(birthday_field, fields.DateField)
+        assert isinstance(person.birthday, date)
 
 
-class DateTimeFieldsFilling(FieldFillingTestCase):
+class TestDateTimeFieldsFilling():
 
-    def test_fill_DateTimeField_with_a_datetime(self):
+    def test_fill_DateTimeField_with_a_datetime(self, person):
         appointment_field = models.Person._meta.get_field('appointment')
-        self.assertIsInstance(appointment_field, fields.DateTimeField)
-        self.assertIsInstance(self.person.appointment, datetime)
+        assert isinstance(appointment_field, fields.DateTimeField)
+        assert isinstance(person.appointment, datetime)
 
 
-class TimeFieldsFilling(FieldFillingTestCase):
+class TestTimeFieldsFilling():
 
-    def test_fill_TimeField_with_a_time(self):
+    def test_fill_TimeField_with_a_time(self, person):
         birth_time_field = models.Person._meta.get_field('birth_time')
-        self.assertIsInstance(birth_time_field, fields.TimeField)
-        self.assertIsInstance(self.person.birth_time, time)
+        assert isinstance(birth_time_field, fields.TimeField)
+        assert isinstance(person.birth_time, time)
 
 
-class UUIDFieldsFilling(FieldFillingTestCase):
-    def test_fill_UUIDField_with_uuid_object(self):
-        uuid_field = models.Person._meta.get_field('uuid')
-        self.assertIsInstance(uuid_field, fields.UUIDField)
-
-        self.assertIsInstance(self.person.uuid, uuid.UUID)
-
-
-@pytest.mark.skipif(connection.vendor != 'postgresql', reason='PostgreSQL specific tests')
-class TestPostgreSQLFieldsFilling:
-    def test_fill_arrayfield_with_empty_array(self, person):
-        assert person.acquaintances == []
-
-    def test_fill_jsonfield_with_empty_dict(self, person):
-        assert person.data == {}
-
-    def test_fill_hstorefield_with_empty_dict(self, person):
-        assert person.hstore_data == {}
+class TestUUIDFieldsFilling():
 
+    def test_fill_UUIDField_with_uuid_object(self, person):
+        uuid_field = models.Person._meta.get_field('uuid')
+        assert isinstance(uuid_field, fields.UUIDField)
+        assert isinstance(person.uuid, uuid.UUID)
 
-class FillingIntFields(TestCase):
 
-    def setUp(self):
-        self.dummy_int_model = mommy.make(models.DummyIntModel)
+class TestFillingIntFields():
 
-    def test_fill_IntegerField_with_a_random_number(self):
+    def test_fill_IntegerField_with_a_random_number(self, db):
```

Comment:
What do you think about adding `@pytest.mark.django_db` to the class instead of inject db every time?

Reply:
Totally agree!